src/index.html

```diff
@@ -7,6 +7,333 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="moyo_logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="is-active nav__link">apple</a>
+          </li>
+          <li class="nav__item">
```

Comment:
Empty lines :harold:
```suggestion
          </li>
          
          <li class="nav__item">
```